content/open-source/herman.rst

```diff
@@ -0,0 +1,27 @@
+public: yes
+location: open-source
+grid_template: true
+image:
+  - src: 'blog/2016/coachhub/feature.jpg'
+project:
+  - name: 'Herman'
+    tagline: 'Automated Style Guides'
+    source: 'https://github.com/oddbird/sassdoc-theme-herman'
```

Comment:
Where is this used? Seems like either this should be `url`, or should be used as a fallback if there's no `url`.